Italian Translations Setup (#708)
italia/eid-wallet-it-docs: docs/it/brand-identity.rst

.. include:: ../common/common_definitions.rst

Brand Identity del Sistema IT-Wallet 
====================================

> La Brand Identity del Sistema IT-Wallet definisce la personalità del sistema e si declina in una serie di elementi codificati che la contraddistinguono, tra cui il naming e gli elementi propri dell’Identità Visiva. Questa contribuisce a rafforzare e promuovere il Modello della Fiducia che sta alla base dell’intero ecosistema ovvero il modello che legittima la partecipazione degli attori al Sistema IT-Wallet e ne determina la sicurezza e l'integrità dei processi. 

Reviewer: ```suggestion
La Brand Identity del Sistema IT-Wallet definisce il naming e gli elementi propri della Identità Visiva. Questa consolida l'implementazione del Modello della Fiducia. 
```

Reviewer: non vogliamo ridefinire cos'è o cosa fa il modello della fiducia

vogliamo ridurre le parole

ogni volta che usiamo "ovvero" significa che non siamo stati chiari nel periodo precedente

Reviewer 2: La brand identity non definisce solo il naming e gli elementi dell'identità visiva, quelli sono solo esempi. D'accordo sulla 2 frase.
```suggestion
La Brand Identity del Sistema IT-Wallet definisce la personalità del sistema e si declina in una serie di elementi codificati che la contraddistinguono, tra cui il naming e gli elementi propri dell’Identità Visiva. Questa consolida l'implementazione del Modello della Fiducia.
```

È quindi importante che la consapevolezza dell’Utente e la sua fiducia nell’ecosistema siano favoriti dalla presenza di elementi grafici, da integrare nelle Soluzioni Tecniche e nei diversi Touchpoint. 

Questa sezione illustra i requisiti minimi che gli Attori Primari devono rispettare per garantire una corretta applicazione della Brand Identity del Sistema IT-Wallet, quindi una sua presenza visiva uniforme, coerente e riconoscibile e un’Esperienza Utente di qualità. 

Per indicazioni e strumenti a supporto, fare riferimento alle Risorse Ufficiali. 

Naming 
------

> “Sistema IT-Wallet" (IT-Wallet System in inglese) o “IT-Wallet" in forma contratta è il nome ufficiale che DEVE essere utilizzato in contesti scritti o verbali, fisici o digitali. Di seguito i relativi requisiti per la corretta scrittura e pronuncia: 

Reviewer: ```suggestion
"Sistema IT-Wallet" (IT-Wallet System in inglese) o "IT-Wallet" in forma contratta è il nome ufficiale che DEVE essere utilizzato in contesti scritti o verbali, fisici o digitali. Di seguito i relativi requisiti per la corretta scrittura e pronuncia: 
```

- si DEVE rispettare l’uso delle maiuscole nel termine “IT” e nelle iniziali di “Wallet” e “Sistema”; 

- si DEVE rispettare l’uso del trattino tra i termini “IT” e “Wallet”, senza l’uso di spaziature; 

- si DEVE rispettare la corretta pronuncia del termine “IT”, separando foneticamente le due lettere “I” e “T”. In italiano la fonetica corretta è /i ti/ e non /it/, mentre in inglese la fonetica corretta è /ˌaɪ ˈtiː/ e non /ɪt/, /aɪt/ o /ɪti/. 
 

Identità Visiva
---------------

> Il Sistema IT-Wallet ha una sua propria un’Identità Visiva. Pertanto, i diversi attori dell’ecosistema DEVONO garantire che la propria Identità Visiva e quella delle proprie Soluzione Tecniche siano in grado di distinguersi e allo stesso tempo di dialogare e coesistere con quella del Sistema IT-Wallet.  

Reviewer: ```suggestion
Il Sistema IT-Wallet ha una propria Identità Visiva. I partecipanti dell’ecosistema DEVONO implementarla.  
```

non vogliamo obbligare gli attori a poterci garantire qualcosa, ma obbligarli a fare qualcosa, come implementare chiaramente qualcosa.

Reviewer 2: il concetto è più complesso: dobbiamo mantenere il vincolo che l'identità visiva del sistema non deve essere applicata come identità visiva di una Soluzione tecnica o di un WP.

```suggestion
Il Sistema IT-Wallet ha una propria Identità Visiva. I partecipanti dell’ecosistema DEVONO applicarla in modo che si distingua e allo stesso tempo dialoghi e coesista con l'Identità Visiva propria e delle proprie Soluzioni Tecniche.  
```

> In particolare, tutti gli Attori Primari POSSONO utilizzare le Risorse Ufficiali relative all’Identità Visiva del Sistema IT-Wallet. Il loro utilizzo DEVE avere come unico scopo quello di rappresentare la partecipazione al Sistema IT-Wallet e non di sostituire dell’Identità Visiva della propria Soluzione Tecnica. 

Reviewer: ```suggestion
In particolare, tutti gli Attori Primari POSSONO utilizzare le Risorse Ufficiali relative all’Identità Visiva del Sistema IT-Wallet. Il loro utilizzo ha lo scopo di rappresentare la conformità al Sistema IT-Wallet senza sostituire l’Identità Visiva della propria Soluzione Tecnica. 
```

non è chiaro che effetti possa avere questo "POSSONO"

Reviewer 2: ```suggestion
In particolare, tutti gli Attori Primari DEVONO utilizzare le Risorse Ufficiali relative all’Identità Visiva del Sistema IT-Wallet. Il loro utilizzo ha lo scopo di rappresentare la partecipazione al Sistema IT-Wallet senza sostituire l’Identità Visiva della propria Soluzione Tecnica. 
```

 
Logo 
^^^^

> Il Logo è l'elemento grafico ufficiale che permette l’immediata riconoscibilità del Sistema IT-Wallet e ne promuove l’affidabilità. 

Reviewer: ```suggestion
Il Logo è l'elemento grafico ufficiale che permette l’immediata riconoscibilità del Sistema IT-Wallet. 
```

Reviewer 2: Cosa non ti torna de "ne promuove l’affidabilità"?

Reviewer: la valutazione dell'affidabilità non credo che avvenga nel dominio grafico, nel senso che un lupo può travestirsi da pecora. Io rimarrei nella riconoscibilità, mentre l'affidabilità deve essere valutata secondo quanto definito nell'apposita sezione relativa alla valutazione della fiducia.

Di seguito i requisiti generali per il suo utilizzo, validi sia in riferimento a contesti di utilizzo fisici che digitali (e.g. siti web, applicazioni, documenti cartacei, materiali informativi stampati o video, etc.): 

- il Logo PUÒ essere utilizzato da tutti coloro che intendono riferirsi al Sistema IT-Wallet; 

- il Logo DEVE essere utilizzato per rappresentare il Sistema IT-Wallet o l’appartenenza ad esso e NON DEVE essere utilizzato per identificare una specifica Soluzione Tecnica; 

- il Logo DEVE essere quello presente all’interno delle Risorse Ufficiali e DEVE seguire le relative specifiche di utilizzo disponibili nelle Risorse Ufficiali; 

> - il Logo DEVE essere utilizzato in formato ``application/svg +xml``; 

Reviewer: ```suggestion
- il Logo DEVE essere utilizzato in formato ``application/svg+xml``; 
```

- il Trust Mark NON DEVE essere alterato, distorto, modificato o sostituito da loghi non ufficiali. 

- il Trust Mark NON DEVE essere ridimensionato oltre i limiti minimi stabiliti dalle Risorse Ufficiali, in modo da garantire sempre una leggibilità ottimale su qualsiasi formato o dispositivo; 

- il Trust Mark DEVE essere utilizzato garantendo l'area di rispetto minima definita nelle Risorse Ufficiali, al fine di garantirne un’adeguata visibilità e riconoscibilità. Altri elementi grafici o testuali NON DEVONO interferire con questa area di rispetto; 

- il Trust Mark NON DEVE essere utilizzato su sfondi di colore che ne compromettano la visibilità o la leggibilità. DEVE essere garantito un contrasto adeguato tra il Logo e lo sfondo, in conformità con quanto definito nelle Risorse Ufficiali; 

- il Trust Mark PUÒ essere associato a loghi, marchi o simboli di altri attori del sistema in accordo con le specifiche di coesistenza, in termini di proporzioni e visibilità, stabilite nelle Risorse Ufficiali. 

Componenti 
^^^^^^^^^^

> I componenti rappresentano quegli elementi del Sistema IT-Wallet che abilitano l’Utente a interagire con le diverse Soluzioni Tecniche tramite la propria Istanza IT-Wallet.  

Reviewer: ```suggestion
Sono definiti componenti gli elementi del Sistema IT-Wallet che abilitano l’Utente a interagire con le diverse Soluzioni Tecniche tramite la propria Istanza IT-Wallet.  
```

Le Risorse Ufficiali mettono a disposizione sia componenti complessi, ovvero template relativi a interi flussi, sia componenti atomici, ovvero singoli elementi da integrare all’interno di interfacce preesistenti (e.g. i Pulsanti di Ingaggio). 

Di seguito i requisiti generali: 

- gli Attori Primari DEVONO utilizzare esclusivamente le Risorse Ufficiali e DEVONO rispettare le indicazioni descritte nelle Risorse Ufficiali; 

- gli Attori Primari POSSONO scegliere quale configurazione, tra quelle rese disponibili, implementare, ma DEVONO comunque garantire il corretto utilizzo dei componenti atomici come i Pulsanti di Ingaggio; 

> - gli Attori Primari DEVONO garantire il costante aggiornamento dei componenti, in linea con l’ultima versione resa disponibile. 

Reviewer: ```suggestion
- gli Attori Primari DEVONO garantire il costante aggiornamento dei componenti, in linea con l’ultima versione resa disponibile. 
```

qui bisogna dire in che tempi

non è possibile pensare che all'immediata emissione di una nuova linea guida, tutti possano adeguarsi immediatamente. Bisogna normare dei tempi.

Reviewer: idealmente, un adeguamento tecnico di media complessità può richiedere fino a 90 giorni

Reviewer: https://github.com/italia/eid-wallet-it-docs/issues/728
